indra/llcharacter/llik.cpp

```diff
@@ -49,6 +49,7 @@ namespace
     constexpr const char *UNKNOWN_CONSTRAINT_NAME               ("UNKNOWN_CONSTRAINT");
     constexpr const char *SIMPLE_CONE_CONSTRAINT_NAME           ("SIMPLE_CONE");
     constexpr const char *TWIST_LIMITED_CONE_CONSTRAINT_NAME    ("TWIST_LIMITED_CONE");
+    constexpr const char *SHOULDER_CONSTRAINT_NAME              ("SHOULDER");
```

Comment:
I'm adding a `ShoulderConstraint` class.  The shoulder joint was previously using a `LimitedTwistCone` but that was insufficient.